pkg/features/kube_features.go

```diff
@@ -195,8 +195,11 @@ const (
 
 	// owner: @aveshagarwal
 	// alpha: v1.9
+	// deprecated: v1.19
 	//
 	// Enable resource limits priority function
+	//
+	// This feature is deprecated, and will be removed in a future release.
```

Comment:
@liggitt we can remove it in this release since it is alpha, right?

Reply:
yes, it can be removed directly. make sure the release note describes the removal and the current alpha state.